src/main/java/com/thebois/models/beings/Colony.java

```diff
@@ -29,30 +24,40 @@
     private final Inventory inventory = new Inventory();
 
     /**
-     * Initializes with already existing beings.
+     * Creates a colony with the pawns that belong to it.
      *
-     * @param beings The beings that should belong to the colony
+     * @param beings Pawns to be associated with the colony.
      */
     public Colony(final Collection<IBeing> beings) {
         setBeings(beings);
     }
 
     /**
-     * Creates an instance of Colony with a number of pawns.
+     * Returns the players' colony.
      *
-     * @param vacantPositions Positions in the world that a Pawn can be placed on.
-     * @param world           The world the pawns move around in.
+     * @return the colony.
      */
-    public Colony(final Iterable<Position> vacantPositions, final IWorld world) {
-        createBeings(vacantPositions, world);
+    public AbstractBeingGroup getColony() {
+        return this;
     }
 
-    private void createBeings(final Iterable<Position> vacantPositions, final IWorld world) {
-        final Random random = new Random();
-        final IPathFinder pathFinder = new AstarPathFinder(world);
-        for (final Position vacantPosition : vacantPositions) {
-            addBeing(new Pawn(vacantPosition, vacantPosition, random, pathFinder));
-        }
+    /**
+     * Returns the players' colony with only inventory methods allowed. (Temporary until we refactor
+     * world/colony)
```

Comment:
Should be updated to reflect the current state of the system. Since this *is* the refactor for world/colony. 😛

Reply:
Yes i just pasted the code from world here :P